frontend/nginx/nginx.conf

```diff
@@ -0,0 +1,53 @@
+server {
+
+  listen 80;
+  
+  location / {
+    root   /usr/share/nginx/html;
+    index  index.html index.htm;
+    try_files $uri $uri/ /index.html;
+
+     #proxy_pass http://192.168.151.107:3030/;
+  }
+
+
+  error_page   500 502 503 504  /50x.html;
+
+  location = /50x.html {
+    root   /usr/share/nginx/html;
+  }
+}
+
+
+upstream backend {
+
+  # nginx as a load balancer with proxy
+  ip_hash;
+  server 192.168.0.1:31300 weight=5;
+  server 192.168.0.1:31301 weight=1;
+  server 192.168.0.1:31302 max_fails=3 fail_timeout=30s weight=1;
+
+}
```

Comment:
Muss sagen bei der nginx config bin ich mir nicht so sicher, ob wir das hier wirklich brauchen - werden aber wahrscheinlich sowieso einen weiteren nginx reverse proxy in der docker-compose haben die letsencrypt und so für uns macht damit wir SSL können.

Reply:
die nginx.conf ist auch zugegebenermaßen einfahc copy paste aus 'nem andern repo, weil ich mich damit nicht groß befassen wollte. Würde es auch erstmal dabei belassen, solange dadurch kein Bug entsteht

Reply:
Jo, denke der Nginx macht dann Port 9000 auf mit Unsinn aber wir geben das eh nicht mit Docker frei, also sollte das passen

Reply:
hab den unnötigen kram nochmal entfernt